core/common/src/main/java/alluxio/conf/RemovedKey.java

```diff
@@ -114,7 +114,7 @@ private static String replacedSince(String version, String newProperty) {
       put("alluxio.user.network.netty.reader.buffer.size.packets", removedSince(V2_0_0));
       put("alluxio.user.network.netty.reader.packet.size.bytes", removedSince(V2_0_0));
       put("alluxio.user.network.netty.timeout", replacedSince(V2_0_0,
-          PropertyKey.USER_NETWORK_DATA_TIMEOUT_MS.getName()));
+          PropertyKey.USER_STREAMING_DATA_TIMEOUT_MS.getName()));
```

Comment:
is this intended?

Reply:
Should I make it `V2_2_3` ?

Reply:
`V2_3_0`, this changes seems big enough to not go into a patch